modules/vaos/app/services/eps/base_service.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module EPS
+  class BaseService < VAOS::SessionService
```

Comment:
As we discussed, we're not gaining anything by inheriting from `VAOS::SessionService`. Rather we should just inherit from `Common::Client::Base`. We should also have a different statsd prefix to not confuse `'api.vaos'` metrics.